docs/README.md

```diff
@@ -1,25 +1,28 @@
+# Architecture Overview
+The InferenceService Data Plane architecture consists of a static graph of components which coordinate requests for a single model. Advanced features such as Ensembling, A/B testing, and Multi-Arm-Bandits should compose InferenceServices together.
 
-# Design Docs
-KFServing architecture consists of two main specifications, around control plane and data plane. Control plane is responsible for deploying and managing the lifecycle of models, while data plane is responsible for defining the contracts to interact with models once deployed.
+![Data Plane](./diagrams/dataplane.jpg)
 
- * [KFserving control plane specification](control-plane.md)
-   * Defines the specification of the `kfserving` resource.
+# Concepts
+**Endpoint**: InferenceServers are divided into two endpoints: "default" and "canary". The endpoints allow users to safely make changes using the Pinned and Canary rollout strategies. Canarying is completely optional enabling users to simply deploy with a BlueGreen deployment stategy on the "default" endpoint.
```

Comment:
While these terms are taken literally, would be good to hyperlink to something which defines BlueGreen etc.